packages/chapters/src/pages/index.md

```diff
@@ -0,0 +1,19 @@
+---
+templateKey: index-page
+hero:
+  title: Debt Collective Chapters
+  subtitle: |-
+    <p class="mt-2">
```

Comment:
I wouldn't do this.
This content is generated automatically after use the CMS.
Therefore adding a class is something that add complexity to the CMS user.

Besides, I strongly recommend to generate this file using the CMS to avoid this temptations and have it formatted as it will be in the future.

Reply:
Yeah I left a comment above regarding this, let's talk about what options we have. Probably I'll move this to CSS since it's something we need just for this page